exercises/concept/annalyns-infiltration/.docs/introduction.md

```diff
@@ -0,0 +1,11 @@
+# Introduction
+
+## Booleans
+
+[Booleans in Rust][booleans-rust-book] are represented by the `bool` type, which values can be either `true` or `false`.
+
+Rust supports three boolean operators: `!` (NOT), `&&` (AND), and `||` (OR).
+
+
+[booleans-rust-book]:
+    https://doc.rust-lang.org/book/ch03-02-data-types.html#the-boolean-type
```

Comment:
We usually don't link to external resources from the introduction. We do link to them via hints and/or the `about.md` of  the concept.

Reply:
Could you also update the `concepts/booleans/introduction.md` and `concepts/booleans/about.md` files?
See https://exercism.org/docs/building/tracks/concept-exercises for more context.